src/plugins/urbanDictionary/index.ts

```diff
@@ -41,12 +49,18 @@ export default definePlugin({
             ],
             execute: async (args, ctx) => {
                 try {
-                    const query = encodeURIComponent(args[0].value);
-                    const { list: [definition] } = await (await fetch(`https://api.urbandictionary.com/v0/define?term=${query}`)).json();
+                    const query: string = encodeURIComponent(args[0].value);
+                    const resultsAmount: number = Settings.plugins.UrbanDictionary.resultsAmount || 10;
+                    const response: Response = await fetch(`https://api.urbandictionary.com/v0/define?term=${query}&per_page=${resultsAmount}`);
+                    const { list } = await response.json();
```

Comment:
```suggestion
                    const { list } = await fetch(`https://api.urbandictionary.com/v0/define?term=${query}&per_page=${resultsAmount}`).then(response => response.json());
```
Feels slightly redundant to bother defining response if you're going to immediately destructure it after.